examples/kotlin/src/main/kotlin/com/xebia/functional/xef/auto/reasoning/WikipediaSearchExample.kt

```diff
@@ -0,0 +1,28 @@
+package com.xebia.functional.xef.auto.reasoning
+
+import com.xebia.functional.gpt4all.getOrThrow
+import com.xebia.functional.xef.auto.ai
+import com.xebia.functional.xef.reasoning.wikipedia.WikipediaClient
+
+suspend fun main() {
+    ai {
+        val client = WikipediaClient()
+
+        val searchData = WikipediaClient.SearchData(
+            search = "Capital%20de%20Colombia"
```

Comment:
This should use URL encoding underneath as we do in #296

Reply:
Done